planets.rb

```diff
@@ -0,0 +1,89 @@
+#Chris Hintz 2.0
+
+class Planets
+
+  attr_accessor :planet,
+                :volume,
+                :mass,
+                :orbit_period,
+                :distance_from_sun
+
+  def initialize(attrs={})
+    @planet             = attrs[:planet]
+    @volume             = attrs[:volume]
+    @mass               = attrs[:mass]
+    @orbit_period       = attrs[:orbit_period]
+    @distance_from_sun  = attrs[:distance_from_sun]
+  end
+end
+
+planets = {
+mercury: Planets.new(
```

Comment:
Incredibly nit-picky suggestion: breaking hashes on separate lines is a great idea, but generally the key/values would be intended one level.